app/views/public/api_documentation.html.erb

```diff
@@ -535,6 +535,7 @@
     "currency": "usd",
     "epoch_mainnet": null,
     "epoch_testnet": null,
+    "epoch_pythnet": null,
```

Comment:
Same as here https://github.com/brianlong/www.validators.app/pull/637/files#r1023762593
If we don't add `epoch_pythnet` column to `sol_prices` table, we don't have to add this information to the doc

Reply:
Ok, I did remove the `epoch_pythnet` entry.